check.sh

```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Why this script is removed?

Reply:
We already have run_all_tests.sh which is running the same command as check.sh along with extra checks. So it was no longer needed.